joomla/joomla-cms
# Change the URL to the PHP Date Function

Current:
http://www.php.net/manual/en/function.date.php - this URL does not work; it requires signing into the php.net site

Proposed:
http://php.net/date - this URL works and is consistent with other language strings MOD_ARTICLES_CATEGORY_FIELD_DATEFIELDFORMAT_DESC and MOD_ARTICLES_CATEGORY_FIELD_MONTHYEARFORMAT_DESC

Pull Request for Issue # .

### Summary of Changes

Current language string:
JGLOBAL_DATE_FORMAT_DESC="Optional format string for showing the date. For example, D M Y for day month year or you can use d-m-y for a short version eg. 28-12-16. See http://www.php.net/manual/en/function.date.php. If left blank, it uses DATE_FORMAT_LC1 from your language file."

Revised language string:
JGLOBAL_DATE_FORMAT_DESC="Optional format string for showing the date. For example, D M Y for day month year or you can use d-m-y for a short version eg. 28-12-16. See http://php.net/date. If left blank, it uses DATE_FORMAT_LC1 from your language file."


### Testing Instructions

(a) Go to the screen "Articles: Options" (Content => Articles > Options)
(b) Click on the tab List Layouts
(c) For the option "Show Date", choose the value Created or Modified or Published)
(d) Move the cursor over the label "Date Format" - Review the pop-up help text
(e) Type the URL shown in the pop-up help text in a separate window/tab of a browser and see what happens

### Expected result

(a) Pop-up Help Text

![after_language_string_change](https://user-images.githubusercontent.com/4947580/48533482-b2774980-e8ca-11e8-9b1c-d280f99b8953.png)

(b) URL to the PHP Date Function taken from the pop-up Help Text

![php_function-date_url_without_www](https://user-images.githubusercontent.com/4947580/48533494-bd31de80-e8ca-11e8-8924-c6d4b275149f.png)


### Actual result

(a) Pop-up Help Text

![before_language_string_change](https://user-images.githubusercontent.com/4947580/48533508-c58a1980-e8ca-11e8-9372-073fcefe9c21.png)

(b) URL to the PHP Date Function taken from the pop-up Help Text

![php_function-date_url_with_www](https://user-images.githubusercontent.com/4947580/48533517-cfac1800-e8ca-11e8-86ff-2cdccec387c2.png)


### Documentation Changes Required
